src/main/java/frc/robot/Constants.java

```diff
@@ -208,7 +209,7 @@ public static final class ElevatorConstants {
     public static final double MAX_HEIGHT_METERS = Units.inchesToMeters(17);
     public static final double STARTING_HEIGHT_METERS = Units.inchesToMeters(2);
     public static final boolean SIMULATE_GRAVITY = true;
-    public static final int ELEVATOR_CURRENT_LIMIT = 30;
+    public static final int ELEVATOR_CURRENT_LIMIT = Robot.modeManager.getMode() == RobotMode.DEMO ? 15 : 30;
```

Comment:
This is something we were playing with to apply demo mode to the elevator acceleration but haven't tested yet. We're also not sure this is the best place to apply this logic